package-lock.json

```diff
@@ -156,7 +156,7 @@
     "babylon": {
```

Comment:
Are any of the changes in package-lock.json required?

Reply:
Removed

Reply:
I still see changes here. If they are not needed, please revert them.